bin/sl-deploy.js

```diff
@@ -62,15 +65,27 @@ if (numArgs > 2) {
   process.exit(1);
 }
 
+var workingDir = process.cwd();
+
+var packageInfo = getPackageInfo(workingDir);
+serviceName = serviceName || (packageInfo ? packageInfo.name : null);
+
+if (!serviceName) {
+  console.error(
+    'Unable to detect service name from package. ' +
```

Comment:
`Unable to detect service name, package.json has no "name" property.` <-- so they know what is missing from the package.